crates/router/src/core/payments/flows.rs

```diff
@@ -166,6 +166,7 @@ default_imp_for_complete_authorize!(
     connector::Coinbase,
     connector::Cryptopay,
     connector::Dlocal,
+    connector::Datatrans,
```

Comment:
Should be in alphabetical order in all the places

Reply:
fixed the required changes